### DISCUSSION TITLE
Apache Airflow credentials change

### DISCUSSION THREAD

**Discussion by Sapstabbs** [open-metadata/OpenMetadata#12062] [Q&A]
Hello! I have successfully installed OpenMetadata on ubuntu following the guideline provided here: "https://docs.open-metadata.org/v1.0.0/quick-start/local-deployment". Then I managed to change the password for my openmetadata admin. However, when I try to change the password for my Airflow admin, Openmetadata cannot connect to it after the credentials change. How can I change my default Apache Airflow password and keep working?

**Comment by herykeven** [comment]
It worked for me by accessing the Airflow UI and creating a new admin user with the same credentials I provided in the openmetadata configuration file
